sum_of_multiples.rb

```diff
@@ -0,0 +1,27 @@
+class SumOfMultiples
+
+  def initialize(*num)
+    @array_of_numbers = num
+  end
+
+  def to(multiple_limit)
+    @result_array = []
+    sum_of_multiples=0
+    (0..@array_of_numbers.length-1).each do |i|
```

Comment:
@HardikPurohit You can loop here directly `@array_of_numbers.each_with_index`, You will get the number inside the array as well as the index as well. You can start the index by a particular number as well.
[Refer Here](http://apidock.com/ruby/Enumerator/each_with_index)